modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -93,7 +93,7 @@ static bool _wait_for_debugger_msecs(uint32_t p_msecs) {
 
 		int tdiff = OS::get_singleton()->get_ticks_msec() - last_tick;
 
-		if (tdiff > p_msecs) {
+		if (tdiff > (int)p_msecs) {
```

Comment:
Instead of casting to `int`, both `tdiff` and `last_tick` should be `uint32_t`, as that is what `get_ticks_msec` returns.

Reply:
Works for me.